### PR TITLE
Add attrs to IconField

### PR DESCRIPTION
**Pull request by davy39**
Fix problem : 'css_class' was not transfered for IconField display.

### CODE REVIEW

File: crispy_bulma/layout.py [hunk 1]
```diff
@@ -227,6 +228,7 @@ def render(
             template=template,
             template_pack=template_pack,
             extra_context=extra_context,
+            attrs=self.attrs
```
Comment (ckrybus) [inline crispy_bulma/layout.py]: a comma at the end is missing

Comment (davy39) [reply crispy_bulma/layout.py]: Sorry for that messy PR... I shouldn't do that directly on github website when tired...
Anyway, that's working for me.
Thanks for your work.

**Comment by ckrybus** [comment]
Thanks for your contribution! :rocket:  I've added a test and simplified it a bit.

File: crispy_bulma/layout.py (hunk 1 above)
Comment (ckrybus) [reply crispy_bulma/layout.py]: @davy39 your fix is available on pypi now - new release 0.8.1, thanks again!

**Pull request merged**